frontend/src/pages/SearchBooks.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {useState} from "react";
+import axios from "axios";
+import {BookAPI} from "../Types/BookAPI";
+import {Box, Container, IconButton, TextField, Typography} from "@mui/material";
+import SearchIcon from "@mui/icons-material/Search";
+
+export default function SearchBooks() {
+    const [search, setSearch] = useState("");
+    const [result, setResult] = useState([]);
+
+    const handleChange = (event: { target: { value: React.SetStateAction<string>; }; }) => {
+        setSearch(event.target.value);
+    }
+
+    const handleSubmit = (event: { preventDefault: () => void; }) => {
+        event.preventDefault();
+        axios.get("https://www.googleapis.com/books/v1/volumes?q=" + search + "&key=AIzaSyC2jzd6VDxU8aIhWzQIWNkZQi4boNbx7Pg&maxResults=20")
```

Comment:
Vorschlag: Wenn du den API Key als env var anlegst, dann kannst du ihn leichter ändern und musst nicht im Code alle Stellen suchen, wo der benutzt wird. 

Um eine ENV var anzulegen: https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env

Benutzen kann man die dann einfach im Code mit `process.env.MY_ENV_VAR`

Reply:
Danke für den Tipp. Muss den Key eh auslagern, weil sonst die Security meckert. 👍🏻